### ISSUE TITLE
*Sunday Work* - Change button to load web menu

### ISSUE THREAD

**Issue by maxmagee** [maxmagee/chicken#9]
The 'View the menu' button on the Sunday menu screen I believe loads the web version of the menu, but I wasn't exactly sure how they implemented that, but it's only viewable on Sunday.

Therefore this has to wait until the next Sunday rolls around.

**Comment by maxmagee** [comment]
The actual application opens the web browser to the web-menu as if it were pushed onto the stack (slides in from the right). The way I'm going to do it below displays it as a modal (which is fine).

```
import { WebBrowser } from 'expo';

handleViewMenuPress = () => {
  WebBrowser.openBrowserAsync('https://www.chick-fil-a.com');
};
```

Might want to consider some error handling here as well.

Here's a direct link to the [documentation](https://docs.expo.io/versions/v29.0.0/sdk/webbrowser).

**Comment by maxmagee** [comment]
Implemented as part of #59
